concordium-consensus/src/Concordium/ProtocolUpdate/P5/ProtocolP6.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- FIXME: This is currently a dummy update, and the details, including the update hash, need to be resolved before release.
+-- TODO: Consider moving the state changes that are made here into the state migration. [This cannot be done after release.]
```

Comment:
What does this TODO refer to?

Which state changes are done here?

Reply:
copy & paste error.